### PR TITLE
create cherry-pick template

### PR DESCRIPTION
**Pull request by itsjustkevin**
Remake of the cherry pick template.

Issue with CLA.

Comments on the original PR can be found at: [99131](https://github.com/flutter/flutter/pull/99131)

### CODE REVIEW

**Review by godofredoc — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/7_cherry_pick.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      options:
+        - "yes"
+        - "no"
+    validations:
+      required: true
+  - type: textarea
+    id: fix_steps
+    attributes:
+      label: Validation Steps
+      description: What are the steps to validate that this fix works?
```
Comment (CaseyHillers) [inline .github/ISSUE_TEMPLATE/7_cherry_pick.yml:71]: Can we ask authors to open a PR as well with their CP to the branch? This lets them fix the merge conflict before it's thrown on the release engineer's plate

Comment (itsjustkevin) [reply .github/ISSUE_TEMPLATE/7_cherry_pick.yml:71]: These are written out in the instructions on the wiki page for cherry picks.  The only issue I see is that they would have to know which branch is the candidate branch.

File: .github/ISSUE_TEMPLATE/7_cherry_pick.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      description: Approximately who will hit this issue (ex. all Flutter devs, Windows developers, all end-customers, apps using X framework feature)?
+      placeholder: All Flutter devs
+    validations:
+      required: true
+  - type: textarea
+    id: impact_description
+    attributes:
+      label: Impact Description
+      description: What is the impact (ex. visual jank on Samsung phones, app crash, cannot ship an iOS app)? Does it impact development (ex. flutter doctor crashes when Android Studio is installed), or the shipping production app (the app crashes on launch)
+      placeholder: Android apps cannot compile without this fix.
```
[the remark this answers is not in the document]
Comment (itsjustkevin) [reply .github/ISSUE_TEMPLATE/7_cherry_pick.yml:35]: You are absolutely correct here, especially for required fields.  Will update.

File: .github/ISSUE_TEMPLATE/7_cherry_pick.yml (hunk 1 above)
Comment (CaseyHillers) [reply .github/ISSUE_TEMPLATE/7_cherry_pick.yml:71]: Good point, I'm not sure. We could skip it in this PR and follow up once there's a better story for that

**Review by CaseyHillers — APPROVED** [review]
flutter-lgtm

**Pull request merged**